### PR TITLE
Add metrics setup instructions and docker files

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

File: dev/metrics/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Readme
+This directory contains some docker files to help you set up metrics tracking on an alluxio cluster. The scripts uses opentelemetry's auto instrumentation java agent to generate tracing information for GRPC and S3 calls, which is visualized using Jaeger and Prometheus. 
+
+1. On one of the Alluxio master or the node where you want to run Jaeger and Prometheus, run 
+```
+docker-compose run -e MASTER_IP=xxx.xx.xx.xx -d
```
Comment (Xenorith) [inline dev/metrics/README.md:6]: to match the same pattern as the worker, rename the master's docker compose file to docker-compose-master.yaml and set the `-f` flag here?

File: dev/metrics/docker-compose-worker.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+version: "2"
+services:
+  # Agent
+  otel-agent:
+    image: otel/opentelemetry-collector-dev:latest
+    command: ["--config=/etc/otel-agent-config.yaml"]
+    volumes:
+      - ./otel-agent-config.yaml:/etc/otel-agent-config.yaml
+    ports:
+      - "8887:8888"   # Prometheus metrics exposed by the agent
```
Comment (Xenorith) [inline dev/metrics/docker-compose-worker.yaml:21]: order ports for ease of searching? ditto for the other port sections

File: dev/metrics/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Readme
+This directory contains some docker files to help you set up metrics tracking on an alluxio cluster. The scripts uses opentelemetry's auto instrumentation java agent to generate tracing information for GRPC and S3 calls, which is visualized using Jaeger and Prometheus. 
+
```
Comment (Xenorith) [inline dev/metrics/README.md:3]: do you have any references/links that you used to figure out all this, especially for the jaeger setup with all the prometheus stuff hooked up?

**Review by Xenorith — CHANGES_REQUESTED** [review]
overall looks good, just some minor comments

**Review by Xenorith — APPROVED** [review]

**Comment by yuzhu** [comment]
```alluxio.server.ft.MasterFailoverIntegrationTest.failoverJournalFencingTest
Error: 3.039 [ERROR]   Run 1: MasterFailoverIntegrationTest.failoverJournalFencingTest:116 » Runtime java.la...
Error: 3.040 [ERROR]   Run 2: MasterFailoverIntegrationTest.failoverJournalFencingTest:116 » Runtime java.la...```

**Comment by yuzhu** [comment]
test failure unrelated

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**